pumpkin-cli/src/main.rs

```diff
@@ -111,13 +111,15 @@ struct Args {
     /// 1-UIP Minimisation is done; according to the idea proposed in "Generalized Conflict-Clause
     /// Strengthening for Satisfiability Solvers - Allen van Gelder (2011)".
     ///
+    /// If this flag is present then the minimisation is turned off.
+    ///
     /// Possible values: bool
     #[arg(
-        long = "learning-minimise",
+        long = "no-learning-minimise",
         default_value_t = true,
```

Comment:
Shouldn't the `default_value_t` here go away?

Reply:
Good point! Should be fixed now